Sources/swift-build/main.swift

```diff
@@ -51,12 +51,18 @@ do {
         let builddir = Path.join(getenv("SWIFT_BUILD_PATH") ?? Path.join(rootd, ".build"), configuration.dirname)
 
         for pkg in dependencies {
-            try llbuild(srcroot: pkg.path, targets: try pkg.targets(), dependencies: dependencies, prefix: builddir, tmpdir: Path.join(builddir, "\(pkg.name).o"), configuration: configuration)
+			try llbuild(srcroot: pkg.path, targets: try pkg.targets(), dependencies: dependencies, prefix: builddir, tmpdir: Path.join(builddir, "\(pkg.name).o"),
+				configuration: configuration,
+				compileExtraArgs: manifest.package.cflags,
+				linkExtraArgs: manifest.package.ldflags)
         }
 
         do {
             // build the current directory
-            try llbuild(srcroot: rootd, targets: targets, dependencies: dependencies, prefix: builddir, tmpdir: Path.join(builddir, "\(pkgname).o"), configuration: configuration)
+            try llbuild(srcroot: rootd, targets: targets, dependencies: dependencies, prefix: builddir, tmpdir: Path.join(builddir, "\(pkgname).o"),
+				configuration: configuration,
+				compileExtraArgs: manifest.package.cflags,
+				linkExtraArgs: manifest.package.ldflags)
```

Comment:
This is somewhat strange style. Perhaps you should make line breaks on each argument, rather than after the first 5?